package/ebs/rancher-ebs

```diff
@@ -350,7 +375,7 @@ delete() {
     local volumes
     volumes=`aws ec2 describe-volumes --region ${EC2_REGION} --volume-ids ${VOLUME_ID} 2>&1`
     if [ $? -ne 0 ]; then
-        if [ "$(echo $error | grep 'InvalidVolume.NotFound')" ]; then
+        if [ "$(echo $volumes | grep 'InvalidVolume.NotFound')" ]; then
             print_success Volume not found
```

Comment:
This will also address https://github.com/rancher/rancher/issues/8607. The problem happens in here is that even the volume is deleted successfully(the error msg have InvalidVolume.NotFound), it will never be considered as succeed and doesn't get cleaned on the docker side.